OpenSim/Tools/CMCTool.cpp

```diff
@@ -86,6 +87,35 @@ CMCTool::CMCTool() :
 }
 //_____________________________________________________________________________
 /**
+ * Default constructor with a specified number of threads (jobs).
+ */
+CMCTool::CMCTool(int numThreads) :
+    AbstractTool(),
```

Comment:
This constructor is not necessary is it? You set the number of threads using the setter.